### PR TITLE
#ab97241 endpoint get site applications

### PR DESCRIPTION
**Pull request by suskiwojciech**
### 🛠 Changes being made

Summarize the changes you've made in this pull request.

### 📸 Screenshots (optional)

Insert some screenshots if you made UI changes.

### 🏎 Quality check

- [ ] Have you performed local tests?
- [ ] Are integration tests passing locally?
- [ ] Have you added some unit tests?
- [ ] Have you added some integration tests?
- [ ] Have you checked WCAG (included screenshot)

### CODE REVIEW

**Review by PawelBargielski — APPROVED** [review]

File: src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs [hunk 1]
```diff
@@ -5,17 +5,225 @@
...
+                var application = _ahpApplicationRepository.GetById(new Guid(applicationId));
+                isLeadPartner = IsConsortiumLeadPartner(consortium, organizationId);
+                isAppPartner = IsApplicationPartner(application, organizationId);
+            }
+
+            if (siteId != null)
+            {
+                TracingService.Trace("Check Access to Site");
+                isLeadPartner = IsConsortiumLeadPartner(consortium, organizationId);
+                isSidePartner = IsOrganizationSitePartner(siteId, organizationId);
```
Comment (jjankowski87) [inline src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs:65]: Side -> Site

File: src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs [hunk 2]
```diff
@@ -5,17 +5,225 @@
...
+                return true;
+            }
+
+            if (isAppPartner && operation == Operation.Get && (recordtype == RecordType.AHPProject
+                || recordtype == RecordType.Application))
+            {
+                return true;
+            }
+            TracingService.Trace("Check web role");
+            UserHasAccess(externalUserId, new Guid(organizationId), siteId, applicationId);
```
Comment (jjankowski87) [inline src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs:94]: return value of this method is not used

File: src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs [hunk 3]
```diff
@@ -5,17 +5,225 @@
...
+            TracingService.Trace("Get Contact");
+            var contact = CrmRepositoriesFactory.Get<IContactRepository>().GetContactViaExternalId(externalUserId);
+            TracingService.Trace("Get Portal Permition Level");
+            List<invln_portalpermissionlevel> ppl = CrmRepositoriesFactory.Get<IPortalPermissionRepository>().GetByAccountAndContact(organizationId, contact.Id);
+            int role = GetRole(ppl);
+            bool accessToAction = false;
+            if (siteId != null)
+            {
+                TracingService.Trace("Check For site");
+                accessToAction = hasUserHavePermitionToProvideOperation(Operation.Get, role, siteId, null, contact.ToEntityReference());
```
Comment (jjankowski87) [inline src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs:166]: Permition -> permission

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs (hunk 1 above)
Comment (suskiwojciech) [reply src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs:67]: Fix

File: src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs (hunk 2 above)
Comment (suskiwojciech) [reply src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs:96]: Fix

File: src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs (hunk 3 above)
Comment (suskiwojciech) [reply src/HE.CRM.AHP.Plugins/Services/Consortium/ConsortiumService.cs]: Fix

**Pull request merged**